tools/harness-thci/OpenThread_BR.py

```diff
@@ -326,10 +324,14 @@ def _deviceBeforeReset(self):
     def _deviceAfterReset(self):
```

Comment:
Do we always call `_deviceAfterReset` even for the first test?

Reply:
Yes, `reset` is being called when adding devices to the setup.

Reply:
SGTM